api/python/quilt3/packages.py

```diff
@@ -245,6 +246,7 @@ def __init__(self):
         self._children = {}
         self._meta = {'version': 'v0'}
 
+    @ApiTelemetry("package.__repr__")
```

Comment:
`Package.__repr__`

Reply:
(I don't really care about lowercase vs uppercase - but we should be consistent.)